src/components/Message/Message.tsx

```diff
@@ -32,6 +32,11 @@ interface PropDefaults {
     onClickMessage: () => void;
     button: false;
 }
+
+/** Test Description
+ *  @category Component
+ *  @constructor Messages
+ */
```

Comment:
Probably should update or remove this.

Reply:
I left that description as I was not sure if the component was 100% working as there was a lot of commented lines. 

My bad, I was going to ask you before I made the PR but forgot lol